### ISSUE TITLE
[BUG] Editing network causes OFF VM to start

### ISSUE THREAD

**Issue by bgulla** [harvester/harvester#791]
**Describe the bug**

If I modify a network on a vm that is currently turned OFF, upon edit, the vm is turned ON.

**To Reproduce**
1. Go to VM
2. Ensure that the VM is off.
3. Edit the network (i.e. masquerade -> bridge)
4. VM automagically turns on 

**Expected behavior**

If a VM is off at time of edit, it should stay off. 

**Environment:**
 - Harvester ISO version: master-d026b5be-head

**Comment by guangbochen** [comment]
@WuJun2016 if the VM is off, I think we are expecting the default option of `Restart now` to be unchecked.
![image](https://user-images.githubusercontent.com/4569037/119474980-46210f00-bd7f-11eb-9fe9-f2650e51b574.png)

**Comment by juadk** [comment]
Tested successfully in build master-6d5c7df2-head ​with ISO mode and a 5 nodes cluster.

I checked that a change in the network settings doesn't start an off VM.
![Screenshot from 2021-06-17 18-01-49](https://user-images.githubusercontent.com/6025636/122434009-040e7600-cf97-11eb-8ba3-442d5da14c68.png)

![Screenshot from 2021-06-17 18-02-16](https://user-images.githubusercontent.com/6025636/122434048-0cff4780-cf97-11eb-9843-f59a480f2038.png)
After saving the new settings the VM stays off as expected.
However, if` Restart the virtual machine now to take effect of the configuration changes.` is checked, the VM is started.

As @guangbochen mentioned, the box is unchecked if the VM is off.
The opposite is true, if the VM is started, the box is checked.

However, when the VM is restarting, I can not see the stopping status, I only see Running, then Starting.
@guangbochen Should I open a new bug for this or it is expected?
